os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.h

```diff
@@ -1086,6 +1100,11 @@ extern u32 rtw_get_tsf(u32 Port);
 WL_BAND_TYPE wifi_get_band_type(void);
 
 
+#if defined(LOW_POWER_WIFI_CONNECT) && LOW_POWER_WIFI_CONNECT

```

Comment:
Why do you check doubled for `LOW_POWER_WIFI_CONNECT`?

Reply:
The first one is for macro definition check and second one is to check whether it has been initialized to 1, i.e. #define LOW_POWER_WIFI_CONNECT 1